test/fixtures/v5/phrase/one_in_distance.json

```diff
@@ -5,7 +5,7 @@
     },
     "phrases": {
         "ar": "بعد 0 units، Do something",
-        "da": "Efter 0 units Do something",
+        "da": "Efter 0 units, Do something",
```

Comment:
There's another problem with this sentence in Danish, unfortunately. If "Do something" is replaced by the arrival string, it becomes "Efter 0 units, Du vil ankomme." or something like that.
The arrival string starts with "Du vil" (You will), but when put after a comma in a sentence like that, it should actually become: "Efter 0 units vil du ankomme", where "du" and "vil" is switched.

This is not a major issue, but an issue nonetheless. This is also true for other sentences added after a distance such as `two_linked_by_distance`.